content/blog/2021-02-10-onboarding-resource-roundup/contents.lr

```diff
@@ -0,0 +1,67 @@
+title: InvenioRDM Onboarding Resource Roundup
+---
+author: Sara Gonzales
+---
+body:
+
+Congratulations! Your team has just joined the InvenioRDM open source project and is eager to contribute as much as possible. With a mixture of dedicated developers, resource curators, subject matter experts, and various leadership stakeholders, your team simply needs to know about the available resources and how to get started. Read on!
+
+
+### Developers
+
+Whether you would like to install and customize InvenioRDM, contribute on the main project sprints, or take on development of a module locally to contribute back to the project, there are a few important points of entry to participation:
+
+- [Roadmap](https://inveniosoftware.org/products/rdm/roadmap/): Familiarize yourself with the overall project timeline and goals. Stable releases will be made through Summer 2021 on a monthly/bi-monthly basis
+
+- [Development Roadmap](https://github.com/inveniosoftware/product-rdm/milestones?direction=asc&sort=due_date&state=open): Get to know the specific milestones for 2021
+
+- [Sprintboard](https://github.com/orgs/inveniosoftware/projects/63): See the planning board for the current sprint
+
+- [Documentation](https://inveniordm.docs.cern.ch/): Read the official project documentation
+
+- [Discord](https://discord.gg/8qatqBC): Ready to get started? Ask Lars for an invitation to the project Discord (chat/video platform) and introduce yourself in either the general or newcomers-help channel. A daily developer standup takes place on Discord at 15:15 UTC
```

Comment:
I am just wondering here if people must go through Discord to "get started"... Something like:  

```suggestion
- [Discord](https://discord.gg/8qatqBC): Ask questions in the project Discord (chat/video platform) and introduce yourself in either the general or newcomers-help channels. A daily developer standup takes place on Discord at 15:15 UTC
```
? This way they don't flood Lars and feel like they can get to work without ceremony ?

Reply:
Sounds good!